lk2nd/device/dts/msm8952/msm8917-mtp.dts

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8917 0>;
+	qcom,board-id = <QCOM_BOARD_ID_MTP 0>;
+};
+
+&lk2nd {
+	xiaomi-ugglite {
+		model = "Xiaomi Redmi Note 5A (ugglite)";
```

Comment:
Could you also add those devices to the devices.md doc?

Reply:
Oh sorry i have forgotten it, ofc.